sass/site.scss

```diff
@@ -54,4 +55,4 @@
 @import "pages/features";
 @import "pages/content";
 @import "pages/migration_guide";
-@import "pages/404";
+@import "pages/404";
```

Comment:
```suggestion
@import "pages/404";

```

Reply:
My SCSS formatter doesn't like this for some reason, I need to configure that